test/runtests.py

```diff
@@ -554,7 +554,7 @@ def main():
     ]
 
     # run each variant
-    pool = Pool() # Use a multiprocessing process Pool
+    pool = Pool(1) # Use a multiprocessing process Pool -- TODO: forcing this to single threaded until we have support for sequential tag
```

Comment:
We can add all sequential tests into another list, and run that list with one single process.